synapse/tests/test_telepath.py

```diff
@@ -122,10 +127,13 @@ def test_telepath_push(self):
 
             self.eq( prox1.bar(10,20), 30 )
 
+            wait0 = env.dmon.waiter(1, 'tele:push:fini')
+
             prox0.fini()
 
-            with self.assertRaises(NoSuchObj) as cm:
-                r = prox1.bar(10, 20)
+            self.nn(wait0.wait(timeout=2))
+
+            self.raises( SynErr, prox1.bar, 10, 20 )
```

Comment:
Do we want to accept any SynErr or NoSuchObj (which is what is raised specifically)?